src/controller/user.controller.js

```diff
@@ -0,0 +1,63 @@
+import User from '../model/user.model.js'
+
+exports.addUser = function (request, response) {
+    const login = request.body.login;
```

Comment:
It is better to use object destructuring here: https://javascript.info/destructuring-assignment#object-destructuring

`const {login, password, age} = request.body`